### PR TITLE
Rewritten the index page

### PR DESCRIPTION
**Pull request by VoyTechnology**
This change kills most of the dead parts in index. Still links to out of date other docs but sections such as Solaris were definitely not needed.

I would be very surprised if this passes any review.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,370 +1,121 @@
...
 
-*  [ Current to Do List](to_do_1502)
+* [Current TODO list](todo)
+* [Suggested Ideas](project-ideas)
 
-*  [Paphos to do list](paphos_migration)
-
-*  [nagios to do list](nagios_to_do)
-
-*  [Suggested Ideas for scripts/projects](project-ideas)
```
Comment (butlerx) [inline docs/index.md]: id like to keep this in

File: docs/index.md [hunk 2]
```diff
@@ -1,370 +1,121 @@
...
+* [redbrick.dcu.ie](redbrick.dcu.ie)
+* [Redbrick SSL certs](ssl)
+* [Mercurial (hg) hosting](hg)
+* [Webchat (qwebirc)](webchat)
 
-*  [Git hosting](git)
-
-*  [Webchat (qwebirc)](webchat)
 #### Backups
+TODO: Add more details as this is definatelly not complete.
```
Comment (butlerx) [inline docs/index.md]: @zergless needs to add backuppc docs

File: docs/index.md [hunk 3]
```diff
@@ -1,370 +1,121 @@
...
-
-*  [Open Ports](openports)
-
-*  [IRC proxy](ircproxy)
-
-*  [Apache configuration on Solaris 10](apachesolaris)
-
-*  [PHP configuration on Solaris 10](phpsolaris)
-
-*  [Apache modules (suphp, pubcookie)](apachemodules)
```
Comment (butlerx) [inline docs/index.md]: id like to keep this in ### www

File: docs/index.md [hunk 4]
```diff
@@ -1,370 +1,121 @@
...
-### UPS
-
-*  [warmstart](warmstart)
-
-*  [electro](electro)
-
-*  [The PDUs](The PDUs)
-### Other
-
-*  [Pike (IP-KVM)](pike)
```
Comment (butlerx) [inline docs/index.md]: is this gone? @CSmartt

File: docs/index.md [hunk 5]
```diff
@@ -1,370 +1,121 @@
...
-
-Docs4 and docs5 are on showboat.theinternets.be and anubis.nowai.org, and mirrored with a [cronjob running on the mirror](docs_mirror_script). These sites are just static html files. They're useful if the main sites aren't available, but images etc. won't work properly, and there may be other issues. The script to generate these mirrors is also a [giant ball of hacks](http://www.laputan.org/mud/mud.html#BigBallOfMud), in particular, only pages linked from this start page will be mirrored.
+---
 
-### CeilingCat, Lightning & Coke
+## System Documentation Redundancy
 
-In case of network failure or other act of god the pages are copied directly from sprout to ceilingcat, lightning & coke as part of /etc/daily.local. There are ssh keys set up so root@sprout can scp to docs@$machine without a password.
+If readthedocs.org goes down, it'd be nice to have access to documentation,
+so we can work on internal things regardless of the network connectivity.
```
Comment (butlerx) [inline docs/index.md]: this doesnt really apply as we dont host readthedocs.org
we do also have them as md files in github for redundancy

File: docs/index.md [hunk 6]
```diff
@@ -1,370 +1,121 @@
...
 
-There's no http server on these machines, but the docs are fairly readable in plain text.
+### docs4.redbrick.dcu.ie
```
Comment (butlerx) [inline docs/index.md]: this is true for docs(2|3|4)

File: docs/index.md [hunk 7]
```diff
@@ -1,370 +1,121 @@
...
-*  [Installing Debian on Cynic](cynicdebian)
-####  Daniel - VM Host
-
-*  [daniel](daniel)
-
-*  [ x4140 ILOM](x4140ilom )
-
-*  [ VM info and password resets](rbvmctl )
+* [Severus (backup server) general setup info](severus)
+* [Moving severus to CSD](severuscolocation)
```
Comment (butlerx) [inline docs/index.md]: @CSmartt keep?

Comment (CSmartt) [inline docs/index.md]: rbvmctl info/resets can stay until a new solution is devised.

File: docs/index.md (hunk 4 above)
Comment (CSmartt) [inline docs/index.md]: ...good question. Leave in for now.

File: docs/index.md (hunk 3 above)
Comment (CSmartt) [inline docs/index.md]: Keep, this is good stuff to keep around in general.

File: docs/index.md (hunk 5 above)
Comment (CSmartt) [inline docs/index.md]: I'd very much like a copy of docs kept locally as well.

File: docs/index.md (hunk 1 above)
Comment (CSmartt) [inline docs/index.md]: Also would like to keep.

File: docs/index.md (hunk 4 above)
Comment (zergless) [inline docs/index.md]: these are still there

File: docs/index.md (hunk 1 above)
Comment (CSmartt) [inline docs/index.md]: This should be kept for now.

File: docs/index.md (hunk 7 above)
Comment (CSmartt) [inline docs/index.md]: Keep for now.

File: docs/index.md (hunk 3 above)
Comment (CSmartt) [inline docs/index.md]: Keep, yep.

File: docs/index.md (hunk 5 above)
Comment (CSmartt) [inline docs/index.md]: Local copy would be nice to have, even if it's just syncing this repo somewhere every week or some such.

File: docs/index.md [hunk 8]
```diff
@@ -1,370 +1,121 @@
...
-
-*  [Webchat (qwebirc)](webchat)
 #### Backups
+TODO: Add more details as this is definatelly not complete.
 
-*  [Redbrick backups with dirvish](Dirvish)
-
-*  [Dirvish Tutorial](dirvish_tutorial)
-
-## Machines
```
Comment (butlerx) [inline docs/index.md]: keep please

File: docs/index.md [hunk 9]
```diff
@@ -1,370 +1,121 @@
...
-
-### Services
```
Comment (CSmartt) [inline docs/index.md]: ...where have services gone?

File: docs/index.md (hunk 6 above)
Comment (VoyTechnology) [inline docs/index.md]: Docs 2 and 3 go to Munin

File: docs/index.md (hunk 1 above)
Comment (VoyTechnology) [inline docs/index.md]: This is still here. https://github.com/redbrick/docs/pull/22/files/d85dc332e7f1a3a22f69c0074cb5b1e70369e52d#diff-1a523bd9fa0dbf998008b37579210e12R13

Comment (VoyTechnology) [inline docs/index.md]: Still here. https://github.com/redbrick/docs/pull/22/files/d85dc332e7f1a3a22f69c0074cb5b1e70369e52d#diff-1a523bd9fa0dbf998008b37579210e12R13

File: docs/index.md (hunk 2 above)
Comment (VoyTechnology) [inline docs/index.md]: This can be done after this PR.

File: docs/index.md (hunk 9 above)
Comment (VoyTechnology) [inline docs/index.md]: Navigation section on the right (once categories are added. I am not sure does it serve much purpose to literally have the same list twice. IMHO index should have links to the major stuff, like day-to-day running, with specific details using search or navigation.

File: docs/index.md (hunk 6 above)
Comment (butlerx) [inline docs/index.md]: they are still live theres no reason to remove them

**Pull request merged**